eng/targetingpacks.targets

```diff
@@ -172,6 +172,18 @@
     </ItemGroup>
   </Target>
 
+  <!-- Don't use the TargetingPack Analyzers for current .NET version unless explicitly enabled -->
```

Comment:
When I read this, it wasn't obvious to me that the target only applies to projects that don't use the local targeting / runtime pack. You want to point that out in the code comment.